tempus_fugit_minecraft/model.py

```diff
@@ -7,7 +7,7 @@
 from pyglet.graphics import TextureGroup, Batch
 from pyglet import image
 from tempus_fugit_minecraft.block import Block,BRICK, STONE, GRASS, SAND, LIGHT_CLOUD,DARK_CLOUD, TREE_TRUNK, TREE_LEAVES
-from tempus_fugit_minecraft.utilities import cube_vertices, WORLD_SIZE, FACES, TICKS_PER_SEC
+from tempus_fugit_minecraft.utilities import cube_vertices, WHOLE_WORLD_SIZE, FACES, TICKS_PER_SEC
```

Comment:
I think we still need to use `WORLD_SIZE` instead of `WHOLE_WORLD_SIZE` as `WHOLE_WORLD_SIZE` is the width of the world from edge to edge and `WORLD_SIZE` (which upon reflection is a bad name) is the distance from origin (0, 0, 0) to an edge on the X,Z plane.

Most of the world gen logic is based on `WORLD_SIZE` anyway, so we should keep using it instead and remove the numbers that are currently the same as or based on `WORLD_SIZE` and replace them with stuff based on the `WORLD_SIZE` constant instead.